src/main/java/seedu/duke/UserInterface.java

```diff
@@ -22,50 +22,72 @@ public void run() {
                 // current implementation is just take 1st value for command
                 String command = userInput.split(" ")[0];
                 switch (command) {
-                case "add":
-                    String regex = "id/(?<id>\\d*) n/(?<name>.*) q/(?<qty>\\d*)"
-                            + " d/(?<desc>\\.*";
-                    Regex regexMatch = new Regex(userInput, regex);
-                    HashMap<String, String> matches = regexMatch.getGroupValues();
-                    Commands.addGood(matches.get("id"), matches.get("name"),
-                            matches.get("qty"), matches.get("desc"), userGoods);
-                    break;
                 case "view":
                     //using flags here to distinguish between different views????
-                    regex = "(?<flag>*[og])/ id/(?<id>\\d*)";
```

Comment:
@tuan0369 i think my old regex might be wrong but this `(?<flag>[og])/` should be right to get the correct flag

Reply:
actually yeah let me change that